src/agent/src/message_task.cpp

```diff
@@ -72,12 +74,14 @@ namespace
 
 } // namespace
 
-boost::asio::awaitable<void> StatefulMessageProcessingTask(std::queue<std::string>& messageQueue)
+boost::asio::awaitable<void> StatefulMessageProcessingTask(const std::string& token,
+                                                           std::queue<std::string>& messageQueue)
 {
-    co_await send_http_request("localhost", "8080", "/stateless", messageQueue);
+    co_await send_http_request("localhost", "8080", "/stateless", token, messageQueue);
```

Comment:
We should also remove these `localhost` and `8080` literals since we have the configparser.

Reply:
Done